spec/book_spec.rb

```diff
@@ -0,0 +1,14 @@
+require './book'
+
+describe Book do
+  book = Book.new('Harry Potter', 'J.K. Rowling')
+  context 'should create instance of book class' do
+    it 'returns correct title' do
+      expect(book.title).to eq('Harry Potter')
+    end
+
+    it 'returns correct author' do
+      expect(book.author).to eq('J.K. Rowling')
+    end
```

Comment:
- [ ] it would be better to add tests for `add_rental` and n order to cover all the methods